instructor_notes/openscpca/openscpca-setup.md

```diff
@@ -0,0 +1,61 @@
+# Instructor Notes: Set up for OpenScPCA
+
+
+## Learning goals
+
+- Understand how to create and launch a Lightsail for Research (LSfR) instance
+- Be prepared to contribute with an OpenScPCA setup, _ideally_ on LSfR (connectivity permitting)
+- Learn differences between local and LSfR setup
+
+## Activity
+
+For this activity, instructors will demonstrate setting up for OpenScPCA contribution and participants should follow along with all steps.
+At the conclusion of this activity, participants should all have a ready-to-go OpenScPCA setup.
+
+The goal for this activity is to set everyone up on LSfR, but we may need to pivot to local setup depending on the internet connectivity.
+Different potential activity options are below:
+
+### Introduction and LSfR instance setup
+
+1. We will begin by presenting a set of slides outlining all setup steps
+   - Slides will also have explained differences between local setup and LSfR setup
+
+2. Direct participants to [fork](https://openscpca.readthedocs.io/en/latest/technical-setup/fork-the-repo/) `OpenScPCA-analysis`
+
+3. Sign into and familiarize with LSfR:
+    - [Launch a (small) VSCodium LSfR instance](https://openscpca.readthedocs.io/en/latest/software-platforms/aws/creating-vcs/).
+      - Explain that we are using VSCodium since we plan to write a python module, but we could launch an RStudio instance for R development as well.
+      Similarly, if we anticipated needing more computational resources, we could select different instance options.
+      - Emphasize the benefits of using volumes when working across several types of instances
```

Comment:
Are we going to demonstrate this? I assume so, and I would lay that out as a separate task here.

Do we talk at all about volume sizes in the docs? This was just a secondary thought I had...

Reply:
Thanks yes, this does need to be its own step. Let's think about scope - 
- we can have them make a volume (noting our [docs](https://openscpca.readthedocs.io/en/latest/software-platforms/aws/working-with-volumes/) only say to pick an appropriate size)
- make an instance with that volume attached
- do we then want to show detaching and attaching to a different instance (eg vscodium -> rstudio)? what do you think? It will add a bit of time but i think it's worth showing, if we can.

Reply:
On size: I think we can suggest something small for this example. (It is _possible_ to expand volumes, and we could potentially add docs about expanding, but I don't want to if we can avoid it)

I don't think we want to show detaching. Let's keep it linear for now.